recipes/rootpath/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "rootpath" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rootpath-{{ version }}.tar.gz
+  sha256: ecc3f9de280ff11c6ade0e956229e4dcb74c1ead5efde8579ba721e6459c518f
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [win]
```

Comment:
I believe this one can be noarch. What is the reason to skip Windows?

Reply:
@ocefpaf thanks for reviewing!

It's due to a missing nested dependancy `[nothing provides blessings needed by colour-runner-0.1.1-py_0](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=838854&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=7c0f8eae-6d6f-51bf-636f-73a1a7fb1bc4)`.  Confirming that there is no [Windows build for Blessings](https://anaconda.org/conda-forge/blessings).

Reply:
Yep, in that case you should make it noarch. Failing on Windows due to a missing dependency is fine. Noarch not only saves CI time but also, then that dependency is available on Windows, nothing is needed in this recipe to make it available there.

Reply:
Thanks @ocefpaf, I'm catching the pattern now. noarch coming up.